main.go

```diff
@@ -308,14 +317,12 @@ func apply(configFile string) (ctrl.Options, config.Configuration, error) {
 		}
 		if len(errorlist) > 0 {
 			err := errorlist.ToAggregate()
-			setupLog.Error(err, "unknown framework", "available", jobframework.GetIntegrationsList())
```

Comment:
Nice work :)
The `field.NotSupported` returns a similar message.

https://github.com/kubernetes/kubernetes/blob/484645e8171930f97a727e28562e8352f579c684/staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go#L200-L213